plugins/cost-insights/README.md

```diff
@@ -222,6 +222,19 @@ costInsights:
       rate: 3.5
 ```
 
+### costChangeNegligibleThreshold (Optional; default 0.5)
+
+This threshold determines whether to show 'Negligible', or a percentage with a fraction of 'engineers' for cost savings or cost excess on top of the charts.  
+A threshold of 0.5 means that `Negligible` is shown when the difference in costs is lower than that fraction of engineers in that time frame,  
+and show `XX% or ~N engineers` when it's above the threshold.
+
+```yaml
+## ./app-config.yaml
+costInsights:
+  engineerCost: 200000
+  engineerThreshold: 0.5
```

Comment:
you should add this to the config.d.ts file as well in the same package, otherwise it won't be readable in the frontend